code/game/objects/items/weapons/defib.dm

```diff
@@ -585,3 +629,17 @@
 					playsound(get_turf(src), 'sound/machines/defib_failed.ogg', 50, 0)
 		busy = FALSE
 		update_icon()
+
+/obj/item/twohanded/shockpaddles/syndicate
+	name = "combat defibrillator paddles"
+	desc = "A pair of mounted high-tech paddles with flat plasteel surfaces to revive deceased operatives(unless they exploded). They possess both the ability to penetrate armor and to deliver powerful or disabling shocks offensively."
```

Comment:
```suggestion
	desc = "A pair of high-tech paddles with flat plasteel surfaces to revive deceased operatives(unless they exploded). They possess both the ability to penetrate armor and to deliver powerful or disabling shocks offensively."
```
These are not mounted onto anything, surely?

Reply:
Technically mounted to the defibrillator but okay.